docs/DeveloperGuide.adoc

```diff
@@ -509,16 +509,21 @@ The calendar mechanism is facilitated by the `DatePickerSkin` skin instance whic
 
 Suppose the address book is modified (e.g. using `add`, `edit`, `addtask`, `deletetask`).
 
-After the modification of the address book i.e. saving of data to the `addressbook.xml` file, `ObservableList` of `Person` and `Task` classes stored in the address book are passed into the `CalendarPanel` Ui.
+After the modification of the address book i.e. saving of data to the `addressbook.xml` file, `ObservableList` of `Person` and `Task` classes stored in the address book are passed into the `CalendarPanel` Ui through `Model`.
```

Comment:
*of Mobilize's address book
put the i.e. saving... in brackets, otherwise it gets hard to figure out where the i.e. ends and the next part begins.